src/librustdoc/html/render/mod.rs

```diff
@@ -1171,8 +1193,8 @@ fn render_assoc_items(
                          <a href=\"#{id}\" class=\"anchor\"></a>\
                      </h2>",
                     id = id,
-                    trait_ = trait_.print(cache, tcx),
-                    type_ = type_.print(cache, tcx),
+                    trait_ = trait_.print(cx.cache(), tcx, relative_to),
+                    type_ = type_.print(cx.cache(), tcx, relative_to),
```

Comment:
Why change this to `cx.cache()`?

Reply:
That was just something I messed up when rebasing.